web3js-ext/src/web3/web3.ts

```diff
@@ -23,7 +23,17 @@ export class KlaytnWeb3 extends Web3 {
     // Override web3.eth.accounts. See web3/src/accounts.ts:initAccountsForContext
     // The functions are bound to 'this' object.
     // TODO: override more web3.eth.accounts methods
+    this.eth.accounts.create = this.accounts_create(this);
+    this.eth.accounts.privateKeyToAccount = this.accounts_privateKeyToAccount(this);
     this.eth.accounts.signTransaction = this.accounts_signTransaction(this);
+    this.eth.accounts.recoverTransaction = this.accounts_recoverTransaction(this);
+    this.eth.accounts.decrypt = this.accounts_decrypt;
+
+    this.eth.accounts.wallet = new Wallet({
+      create: this.eth.accounts.create, 
+      privateKeyToAccount: this.eth.accounts.privateKeyToAccount,
+      decrypt: this.accounts_decrypt,
```

Comment:
Consistency with the lines right above it.
```suggestion
      decrypt: this.eth.accounts.decrypt,
```

Reply:
Updated in https://github.com/klaytn/web3klaytn/pull/522/commits/c768100af9e0744ec7bda1ad413aa573e6851697. 

Type check return 
"Type '(keystore: string, password: string, options?: Record<string, unknown> | undefined) => Promise<Web3Account>' is not assignable to type '(keystore: string | KeyStore, password: string, options?: Record<string, unknown> | undefined) => Promise<Web3Account>'." 

But there is no problem because Our function can accept KeyStore type. 
So I just add "// @ts-ignore." line.